src/java/picard/analysis/CollectBaseDistributionByCycle.java

```diff
@@ -19,13 +19,30 @@
 import picard.cmdline.programgroups.Metrics;
 import picard.util.RExecutor;
 
+
+
 @CommandLineProgramProperties(
-        usage = "Program to chart the nucleotide distribution per cycle in a SAM or BAM file.",
+        usage = "Program to chart the nucleotide distribution per cycle in a SAM or BAM file.  In addition to charting" +
```

Comment:
Would probably be helpful to say what we're expecting here and how to interpret results: I believe (but we should check) that in general, we expect that for any given cycle, or position within reads, the relative proportions of A, T, C and G should reflect the AT:GC content of the organism's genome. So we expect flattish lines for all four nucleotides. If we see anything that deviates from this expectation, for example a spike of A at a particular cycle (position within reads) then that suggests a systematic sequencing error